src/cargo.rs

```diff
@@ -1,15 +1,15 @@
+use rand::distributions::{ChiSquared, Distribution, Exp};
 /// Module that pretends to run cargo to install rust packages.
```

Comment:
I suppose this should be a `//!` right?

Reply:
there was no update made to this line `/// Module that pretends to run cargo to install rust packages.` by `cargo fmt`

Reply:
Well, it was re-arranged.

Reply:
I will check if there is any config related to `module docstrings` in rustfmt.toml and reply by this weekend

Reply:
I think `///` is correct. I tried updating to `//!` and ran `cargo check` I got - 
```error: expected outer doc comment
 --> src/cargo.rs:2:1
  |
2 | //! Module that pretends to run cargo to install rust packages.
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: inner doc comments like this (starting with `//!` or `/*!`) can only appear before items

error: aborting due to previous error

error: could not compile `genact`.
```